parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -151,7 +150,7 @@ if [[ "${GPU_NODE}" = true ]] && [[ "${skip_nvidia_driver_install}" != "true" ]]
 [Service]
 Environment="MIG_STRATEGY=--mig-strategy single"
 ExecStart=
-ExecStart=/usr/local/nvidia/bin/nvidia-device-plugin $MIG_STRATEGY    
+ExecStart=/usr/local/nvidia/bin/nvidia-device-plugin $MIG_STRATEGY
```

Comment:
be careful about trimming spaces here

Reply:
⁠did not get this comment, don't see any concatenation/append to the command, so removing space should be ok? going to undo changes for spaces/trims but just trying to understand the issue.